src/Raven.Server/Documents/ETL/Providers/Queue/Kafka/KafkaEtl.cs

```diff
@@ -1,27 +1,34 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
 using CloudNative.CloudEvents;
 using CloudNative.CloudEvents.Kafka;
 using Confluent.Kafka;
+using Confluent.Kafka.Admin;
 using Raven.Client.Documents.Operations.ETL;
 using Raven.Client.Documents.Operations.ETL.Queue;
+using Raven.Client.Util;
 using Raven.Server.Documents.ETL.Stats;
 using Raven.Server.Exceptions.ETL.QueueEtl;
 using Raven.Server.ServerWide;
 using Raven.Server.ServerWide.Context;
+using ResourceType = Confluent.Kafka.Admin.ResourceType;
 
 namespace Raven.Server.Documents.ETL.Providers.Queue.Kafka;
 
 public class KafkaEtl : QueueEtl<KafkaItem>
 {
     private IProducer<string, byte[]> _producer;
+    internal string TransactionalId { get; }
 
     public KafkaEtl(Transformation transformation, QueueEtlConfiguration configuration, DocumentDatabase database, ServerStore serverStore) : base(transformation, configuration, database, serverStore)
     {
+        TransactionalId = $"{Database.ServerStore.GetClusterTopology().TopologyId}-{Name}".Replace("/", "_"); // characters '/' don't pass the validation - we replace them
```

Comment:
@poissoncorp we need a database topology id not cluster topology id

Reply:
fixed ba1813e0f6b4b4037524bc32f1adffff1a21c680